src/main/scala/ru/ledostuff/calories/infrastructure/services/ProductCaloriesService.scala

```diff
@@ -42,23 +42,23 @@ class ProductCaloriesService[F[_]: Sync](productService: ProductService[F],
     ).getOrElseF({
       val resultOrError: EitherT[F, ErrorResponse, ProductCaloriesInfo] = for {
 
+        _                     <- EitherT.liftF(log.info(s"search translation for name $name"))
         translatedProductInfo <- EitherT.fromOptionF[F, ErrorResponse, TranslatedProduct]({
-          (OptionT.liftF(log.info(s"search translation for name $name")) *>
-          translateService.translateProductName(name)).value
+          translateService.translateProductName(name).value
         }, TranslationNotFound)
 
-        savedProduct          <- EitherT.liftF[F, ErrorResponse, Product](productService.createProduct(Product(name, translatedProductInfo.translatedNames.toSeq)))
+        savedProduct          <- EitherT.liftF[F, ErrorResponse, Product](productService.createProduct(Product(name, translatedProductInfo.translatedNames.toList)))
 
         caloriesInfo          <- {
-          val engNamesRequests = savedProduct.engNames.toList.map { engName =>
+          val engNamesRequests = savedProduct.engNames.map { engName =>
             (EitherT.liftF[F, ErrorResponse, Unit](log.info(s"search calories for name $engName")) *>
```

Comment:
I wanted you to make it everywhere :) Perhaps you miss it

Reply:
Nonono :) I am calling log before each request to the calories service